docs/news.d/config_store.rst

```diff
@@ -1,4 +1,4 @@
-.. news-prs: 4162 4241 4242 4255 4243 4249
+.. news-prs: 4162 4241 4242 4255 4243 4249 4272 4270
 .. news-start-section: Additions
 - OpenDDS now stores ``rtps_udp`` transport configuration in the key-value store.
```

Comment:
Could all of these combined into one item? Also could you like to the documentation of the config store so users can see what that is?